public/data/wasiqb.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "Wasiq Amjad Bhamla",
+  "type": "personal",
+  "bio": "Open source enthusiast | Senior QA Consultant at Thoughtworks",
+  "avatar": "https://github.com/WasiqB.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/WasiqB",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/WasiqBhamla",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/wasiqbhamla",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started GitHub account",
+      "date": "2014",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Opened GitHub account without knowing about anything about open source",
+      "url": "https://github.com/WasiqB"
+    },
+    {
+      "title": "coteafs-appium",
+      "image": "https://github.com/WasiqB.png",
+      "date": "April 2017",
+      "icon": "github",
+      "color": "black",
+      "description": "My first serious open source project for Mobile Automation testing",
+      "url": "https://github.com/WasiqB/coteafs-appium"
+    },
+    {
+      "title": "coteafs-selenium",
+      "date": "April 2018",
+      "icon": "github",
+      "color": "green",
+      "description": "Open sourced my 2nd project for Web automation testing",
+      "url": "https://github.com/WasiqB/coteafs-selenium"
+    },
+    {
+      "title": "AppiumTestDistribution",
+      "date": "May 2019",
+      "icon": "github",
+      "color": "blue",
+      "description": "My first contribution to other project",
```

Comment:
```suggestion
      "description": "My first contribution to a project",
```

Reply:
@krishguptadev , This was my first contribution to other peoples open source project. I had already contributed to my own projects. Hence, mentioned `other project`.

Reply:
It would be ... ig

```suggestion
      "description": "My first contribution to another project",
```